.github/CODEOWNERS

```diff
@@ -16,14 +16,7 @@
 ##################
 # Catch all
 ##################
-*                                      @jongio @conniey @g2vinay @hallipr
-
-##################
-# Documentation
-##################
-/docs/                                 @jongio @rohit-ganguly
-LICENSE                                @jongio @rohit-ganguly
-*.md                                   @jongio @rohit-ganguly
+*                                      @Azure/azure-mcp
```

Comment:
@jongio for my context what is the goal of this github team? Should it be a child of our standard azure-sdk-write team?